bigframes/session/__init__.py

```diff
@@ -377,6 +378,10 @@ def slot_millis_sum(self):
         """The sum of all slot time used by bigquery jobs in this session."""
         return self._slot_millis_sum
 
+    @property
+    def allow_ambiguity(self) -> bool:
```

Comment:
Does this need to be a public property? I'd prefer to keep it private until we have a request from users to modifying it.

Also, I'm not sold on the naming. Should it be specific and mention ambiguous orderings? Or rename to allow all forms of non-determinism?

Reply:
Yes, no reason to make this a public property. This is specifically for use in windowing - where ambiguous orderings produce ambiguous row values. Ordering ambiguity is introduced through "partial" ordering mode already.